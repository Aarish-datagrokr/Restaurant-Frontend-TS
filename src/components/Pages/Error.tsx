import "./styles/error.css";
const Error = () => {
    return (
        <div className='error-container'>
            <img className="image" src="https://cdn.dribbble.com/users/492116/screenshots/1667059/thrillist-404.gif"></img>
            <h1 className="error-message">PAGE NOT FOUND!</h1>
        </div>
    );
}

export default Error;