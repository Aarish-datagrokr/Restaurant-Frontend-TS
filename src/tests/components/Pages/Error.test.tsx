import {shallow} from 'enzyme';
import Error from '../../../components/Pages/Error';

test('Should test Error component', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
});