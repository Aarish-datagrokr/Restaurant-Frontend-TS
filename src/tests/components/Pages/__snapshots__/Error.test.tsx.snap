// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test Error component 1`] = `
<div
  className="error-container"
>
  <img
    className="image"
    src="https://cdn.dribbble.com/users/492116/screenshots/1667059/thrillist-404.gif"
  />
  <h1
    className="error-message"
  >
    PAGE NOT FOUND!
  </h1>
</div>
`;
